org.lflang/src/org/lflang/TargetConfig.java

```diff
@@ -260,6 +260,10 @@ public class TargetConfig {
      */
     public boolean exportToYaml = false;
 
+    public boolean microSecondGranularity = false;
+
```

Comment:
Note: These don't have any purpose as of yet as we would need to add configurations for each of these values. Can be apart of a larger commit bringing 32 Bit/microsecond granularity support to Reactor-C